server/.env.example

```diff
@@ -12,6 +12,8 @@ NEO4J_ENTERPRISE=false
 
 JWT_AUD=
 
+TRIPLER_SEARCH_NAME_BOOST=
```

Comment:
could you add this and FRAUD_THRESHOLD to the README as well?

Reply:
I just did :)